src/Model/GalleryInterface.php

```diff
@@ -42,57 +42,51 @@ public function getContext();
     /**
      * @param string $context
      *
-     * @return void
+     * @return static
```

Comment:
why change to fluent interface?

Reply:
I had to change some of the Models phpdoc to make phpstan happy, then I realize we weren't using fluent interface for entities. I tried to be consisent on how the maker-bundle generates entities, because that will be how most of our users will expect the entities to be.

Do you prefer doing this change on a separate PR or you don't like fluent interfaces?

Reply:
Well, I don't like fluent interfaces for mutable objects (except for builders), I try to avoid them, not sure if they make sense in the interfaces for models.

Ref: https://ocramius.github.io/blog/fluent-interfaces-are-evil/

Reply:
I will revert those changes because they are not needed to bump phpstan. IMO fluent entity is not a big deal but no strong feelings to not have them.

Reply:
Revert